includes/abstracts/class-wc-gateway-mondido-abstract.php

```diff
@@ -388,11 +394,13 @@ public function handle_transaction( $order, $transaction_data ) {
                 'postcode'   => $details['zip'],
                 'country'    => $this->get_country_alpha2( $details['country_code'] ),
             );
-            update_post_meta( $order_id, '_mondido_invoice_address', $address );
-        }
+		}
```

Comment:
Shouldn't this bracket come after the call to update and save?

Reply:
Yes

Reply:
Now it seems like the indentation is wrong here. Or is it just the display in GitHub?